database/seeders/DatabaseSeeder.php

```diff
@@ -13,11 +14,49 @@ class DatabaseSeeder extends Seeder
      */
     public function run(): void
     {
-        // User::factory(10)->create();
-
-        User::factory()->create([
-            'name' => 'Test User',
-            'email' => 'test@example.com',
-        ]);
+        $users = [
+            [
+                'name' => 'Pasha',
+                'email' => 'www@PashkaPASS555777',
+                //         'password' => '555777',
+                'email_verified_at' => NULL,
+                // hash ниже
+                'password' => '$2y$12$px9y5AgGxGjSOMKZ9vATN.6dDSYJ0UV6gsDns8YHbdEsLmcKSHd0W',
```

Comment:
create a static PasswordHashGenerator which must generate a hashed password dependent on the user's password